test/helpers/definitions.js

```diff
@@ -80,7 +82,8 @@ const SHOPPING_CART_SERVICE = {
       ],
       returnType : { name: 'Promise', elementType: { name: 'Order' } }
     }
-  }
+  },
+  serviceInfo: {}
```

Comment:
My changes do not affect tests, but I decided to make them succeded

Reply:
great! thanks